classes/help.php

```diff
@@ -25,11 +25,10 @@ function print_page() {
 				$simple_map->show_toolbar( $sm_page_title );
 				?>
 
-				<div><p><?php _e( 'Jump to a section:', 'SimpleMap' ); // XSS ok. ?> <a
-							href="#displaying_your_map"><?php _e( 'Displaying Your Map', 'SimpleMap' ); // XSS ok. ?></a> | <a
-							href="#general_options"><?php _e( 'General Options', 'SimpleMap' ); // XSS ok. ?></a> | <a
-							href="#adding_a_location"><?php _e( 'Adding a Location', 'SimpleMap' ); // XSS ok. ?></a> | <a
-							href="#everything_else"><?php _e( 'Everything Else', 'SimpleMap' ); // XSS ok. ?></a></p></div>
+				<div><p><?php _e( 'Jump to a section:', 'simplemap' ); // XSS ok. ?> <a
+							href="#displaying_your_map"><?php _e( 'Displaying Your Map', 'simplemap' ); // XSS ok. ?></a> | <a
+							href="#general_options"><?php _e( 'General Options', 'simplemap' ); // XSS ok. ?></a> | <a
+							href="#adding_a_location"><?php _e( 'Adding a Location', 'simplemap' ); // XSS ok. ?></a></p></div>
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 127 characters

<!-- Comment by [SideCI](https://sideci.com) -->